### PR TITLE
Remove outdated save template in favor of default creation.

### PR DESCRIPTION
**Pull request by apachano**
## Changes
Update saves system to create entire path if Saves folder is absent
Remove template save in-favor of game driven creation

## On approval
Merge

### CODE REVIEW

File: Phoenix/Server/Source/Game.cpp [hunk 1]
```diff
@@ -38,13 +38,19 @@ using namespace phx;
...
     : m_blockRegistry(blockReg), m_registry(registry), m_iris(iris),
       m_map(voxels::Map(save, "map1", &blockReg->referrer))
 {
 	m_commander = new Commander(m_iris);
 }
 
+Game::~Game()
+{
+	if (m_running){kill();}
+    delete m_commander;
```
Comment (SuperFola) [inline Phoenix/Server/Source/Game.cpp:51]: Indentation (this and the line above)

**Comment by beeperdeeper089** [comment]
What fola said, but i'm fine with this once that's fixed.

File: Phoenix/Server/Source/Game.cpp (hunk 1 above)
Comment (tobyplowy) [reply Phoenix/Server/Source/Game.cpp:51]: Also tabs and spaces (Line 51).

**Comment by apachano** [comment]
Fixed, could I get a new stamp?

**Pull request merged**